internal/converter/openapi_mlmd_converter_util.go

```diff
@@ -114,8 +114,12 @@ func PrefixWhenOwned(ownerId *string, entityName string) string {
 	} else {
 		prefix = uuid.New().String()
 	}
-	prefixedName := fmt.Sprintf("%s:%s", prefix, entityName)
-	return prefixedName
+
+	if entityName == "" {
+		return prefix
+	}
```

Comment:
great catch, although I'm now wondering if we should really allow nameless logical model entities.